README.md

```diff
@@ -30,6 +30,8 @@ The library provides both synchronous (blocking) API calls, as well as asynchron
 ## API overview
 
 - [dronecore](include/dronecore.h): set up connection, discover devices
+- [device](include/device.h): a device providing access to modules below using ...
+- [device_plugin_container.h.in](include/device_plugin_container.h.in) which is auto-generated on build.
```

Comment:
@julianoes "Which is auto-generated on build". Does that mean it can be ignored by everyone and has no point? Or is this essential, and you need to know about it?  Upshot, a useful comment should say what something is for.

E.g. 
[device_plugin_container.h.in](include/device_plugin_container.h.in): Header that is auto-generated on build. Used to blah blah blah.